lti/sentry.py

```diff
@@ -0,0 +1,19 @@
+"""Report exceptions to Sentry."""
+
+import os
+
+import raven
+
+
+def get_raven_client(request):
+    """Return the Raven client for reporting crashes to Sentry."""
+    client = request.registry["raven.client"]
+    client.http_context({"url": request.url, "method": request.method})
+    request.add_finished_callback(lambda request: client.context.clear())
+    return client
+
+
+def includeme(config):
+    environment = os.environ.get('ENV', 'dev')
+    config.registry["raven.client"] = raven.Client(environment=environment)
```

Comment:
Do we need to add some sanitizers to this as well? https://github.com/hypothesis/h/blob/master/h/sentry.py#L17-L20